policyengine_us/variables/gov/states/mt/tax/income/mt_income_tax_before_refundable_credits.py

```diff
@@ -0,0 +1,35 @@
+from policyengine_us.model_api import *
+
+
+class mt_income_tax_before_refundable_credits(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Montanta income tax before refunable credits"
```

Comment:
Montana typo elsewhere
```suggestion
    label = "Montana income tax before refundable credits"
```